TRANSLATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Translations
+
+The HTML 5.1 specification is now a stable document. We invite you to [volunteer to translate](https://www.w3.org/Consortium/Translation/) this specification, and help people around the world to reach the HTML 5.1 project.
+
+## How to Build
+
+The specification is built using [Bikeshed](https://github.com/tabatkins/bikeshed). If you would like to take advantage of the single-page.bs and build the spec when you've completed your translation, here are some suggestions for you:
+
+1. Fork the w3c/html repo
+2. Clone your fork to local: `git clone https://github.com/yourfork/html`
+3. Checkout the html5.1 branch: `git checkout html5.1`
+4. Translate the documents
+5. Install [bikeshed](https://github.com/tabatkins/bikeshed)
+6. From the HTML folder open a command prompt
+7. Run bikeshed update: `bikeshed update`
+8. Run bikeshed: `bikeshed spec`
+
+Now you have a single-page.html, you may wish to build the multipage version, please do as follows:
+
+1. Install [multipage](https://github.com/adrianba/multipage)
```

Comment:
For me, and I guess that for @mulder21c the "Install [multipage](https://github.com/adrianba/multipage)" does not work as expected.

With proposed changes from http://blog.publisher.name/, I had to change `.multipage-split.sh` file to a fork to be able to push. See this commit https://github.com/fititnt/multipage/commit/24f620adcf213577c07f48feda55fc8b281597b2